helpers.js

```diff
@@ -1,19 +1,33 @@
 const util = require("util");
 const childProcess = require("child_process");
-const { access } = require("fs/promises");
-const fs = require("fs");
+const fs = require("fs/promises");
 
 const exec = util.promisify(childProcess.exec);
 
 async function assertPathExistence(path) {
   try {
-    await access(path, fs.constants.F_OK);
+    await fs.access(path, fs.constants.F_OK);
+    return true;
   } catch {
-    throw new Error(`Path ${path} does not exist`);
+    return false;
+  }
+}
+
+async function getFileContent({ PATH: path }) {
+  if (assertPathExistence(path)) {
```

Comment:
Forgot the `await` before calling `assertPathExistence`.

Reply:
how embarrassing.